docs/llm/reference.md

```diff
@@ -130,12 +163,14 @@ When default template is loaded, servable accepts `/chat/completions` calls when
 
 ## Limitations
 
-As it's in preview, this feature has set of limitations:
+LLM calculator is a preview feature. It runs a range of accuracy, stability and performance test but the next releases targets production grade quality. It has now a set of known issues:
 
-- Limited support for [API parameters](../model_server_rest_api_chat.md#request),
-- Only one node with LLM calculator can be deployed at once,
-- Metrics related to text generation - they are planned to be added later,
-- Improvements in stability and recovery mechanisms are also expected
+- Metrics related to text generation are not exposed via `metrics` endpoint. Key metrics from LLM calculators are included in the server logs with information about active requests, scheduled for text generation and KV Cache usage. 
+- Models using in the template empty bos_token require updating the tokenizer config with a command: `sed -i '/"bos_token": null,/d' tokenizer_config.json` The known models which require such workaround are `Qwen1.5-7B-Chat` and `allenai/OLMo-1.7-7B-hf`. It won't be needed in the next release. This issue is not impacting `completions` endpoint.
+- llama3.1 models observe accuracy issues and overlong responses - this is investigated.
+- in rare cases when the model generates non valid utf8 sequence, it will be returned to the client without replacing it with � unicode replacement character. Use this code `text.decode("utf-8",errors='replace')` to make the replacement on the client side.
```

Comment:
```suggestion
- In rare cases when the model generates non valid utf8 sequence, it will be returned to the client without replacing it with � unicode replacement character. Use this code `text.decode("utf-8",errors='replace')` to make the replacement on the client side.
```